internal/cloud/realm/import_export_test.go

```diff
@@ -319,46 +319,40 @@ func appDataV2(app realm.App) local.AppDataV2 {
 		Sync: &local.SyncStructure{
 			Config: map[string]interface{}{"development_mode_enabled": true},
 		},
+		Functions: &local.FunctionsStructure{
+			Configs: []map[string]interface{}{
+				{"name": "test", "private": true},
+			},
+			Sources: map[string]string{
+				"test.js": `exports = function(){
+		  console.log('got heem!');
+		};`,
+			},
+		},
+		Triggers: []map[string]interface{}{
+			{
+				"name":          "yell",
+				"type":          "SCHEDULED",
+				"config":        map[string]interface{}{"schedule": "0 0 * * 1"},
+				"function_name": "test",
+				"disabled":      false,
+			},
+		},
```

Comment:
[minor q] Do database triggers have a significantly different structure as scheduled triggers (I imagine the type and config would be the differences here but not sure)? I wonder if it's worth testing one here as a sanity test

Reply:
it was easy enough to add